File: local/HDR__/c_src/factorial2.c
#include <stdio.h>

long long factorial(int n)
{
	long long fact = 1;
	if (n == 1)
		return 1LL;
	else
		return factorial(n-1);
}

int main(void)
{
	for (int i = 1; i <= 100; ++i){
		long long result = factorial(i);
		printf("%d! : %lld\n", i, result);
	}

	return 0;
}

File: local/HDR__/python/factorial2.py
def factorial(n):
    if n == 1:
        return 1
    else:
        return n * factorial(n - 1)
    

result = [factorial(n) for n in range(1, 100+1)]
result = list(map(factorial, [n for n in range(1, 100+1)]))
print(result)